website/_i18n/ar/user-guides/run-pactus-metrics/index.md

```diff
@@ -0,0 +1,136 @@
+---
+version: 1
+---
+
+## Preface
+
+Pactus can be run with metrics, providing you with the ability to monitor your node.
+This tutorial will guide you through the steps to run Pactus with metrics, suitable for advanced users.
+
+## Prerequisites
+
+Before proceeding with the steps below, ensure that you have the following:
+
+- An understanding of Prometheus and Grafana.
+- Have a running Pactus Node.
```

Comment:
Have a Pactus node synced with the network.

Reply:
we don't need to a synced node for metrics.